fuzzaldrin-plus/index.d.ts

```diff
@@ -5,25 +5,32 @@
 
 export as namespace fuzzaldrin;
 
-export interface IQueryOptions {
+export interface QueryOptions {
```

Comment:
@JKillian, I understand your point but my understanding is that _types-2.0_ branch is meant for future typings (that they would be merged to master) and only the ones in _master_ is available to TSD. Is there a distinction here on when should we put something in _master_ vs other branches in this repository? Are Devs taking dependency on typings available on non-master branches?